components/kyma-environment-broker/internal/storage/dbsession/write.go

```diff
@@ -73,10 +75,19 @@ func (ws writeSession) UpdateInstance(instance internal.Instance) dberr.Error {
 		Set("provisioning_parameters", instance.ProvisioningParameters).
 		Set("provider_region", instance.ProviderRegion).
 		Set("updated_at", time.Now()).
+		Set("version", instance.Version+1).
 		Exec()
 	if err != nil {
 		return dberr.Internal("Failed to update record to Instance table: %s", err)
 	}
+	rAffected, e := res.RowsAffected()
```

Comment:
i think there is no need to change the error naming convention, we can overwrite previous `err` value here
```suggestion
	rAffected, err := res.RowsAffected()
```

Reply:
done